ts/odk-ts/ow-window.ts

```diff
@@ -84,15 +84,15 @@ export class OWWindow {
   public async getWindowState(): Promise<GetWindowStateResult> {
     let that = this;
 
-    return new Promise(async resolve => {
+    return new Promise<GetWindowStateResult>(async resolve => {
       await that.assureObtained();
       let id: string = <string>that._id;
       overwolf.windows.getWindowState(id, resolve);
     })
   }
 
   public static async getCurrentInfo(): Promise<overwolf.windows.WindowInfo> {
-    return new Promise(async resolve => {
+    return new Promise<overwolf.windows.WindowInfo>(async resolve => {
```

Comment:
Please add a type for it, as seen in line 1

Reply:
@omerschapiro Sure I'll do it today or tomorrow :)